### PR TITLE
feat(nutanix): add VLAN ID field in cluster redeployment

### PR DESCRIPTION
**Pull request by ThibaudCrespin**
| Question         | Answer
| ---------------- | ---
| Branch?          |  `develop` 
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-14545 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~Breaking change is mentioned in relevant commits~

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/translations/Messages_de_DE.json [hunk 1]
```diff
@@ -8,5 +8,6 @@
...
   "dedicatedCloud_datacenter_EOL_title": "Ablaufdatum der Dedicated Cloud: 30. September 2022",
   "dedicatedCloud_datacenter_EOL_part_1": "Im Oktober 2021 hatten wir bereits angekündigt, dass unsere Angebote der Art Dedicated Cloud auslaufen und am 30. September 2022 endgültig eingestellt werden. Dieser Schritt ist im Hinblick auf Ihre VMware-Plattformen von Bedeutung, da damit eine Verbesserung der Dienstqualität, der Performance und der Sicherheit Ihrer Cluster einhergeht. Die Hardware, die vor 2018 für die Angebote vom Typ „Dedicated Cloud“ verwendet wurde, ist nicht dazu geeignet, den höchsten Service-Level nach heutigem Stand zu gewährleisten. Außerdem ist keine Integration mit den neuen VMware-Architekturen möglich.",
-  "dedicatedCloud_datacenter_EOL_part_2": "Zahlreiche Kunden haben diese Migration bereits geplant und begonnen. Wir möchten dies nun auch Ihnen noch vor dem Stichtag am 30. September 2022 empfehlen."
+  "dedicatedCloud_datacenter_EOL_part_2": "Zahlreiche Kunden haben diese Migration bereits geplant und begonnen. Wir möchten dies nun auch Ihnen noch vor dem Stichtag am 30. September 2022 empfehlen.",
+  "dedicatedCloud_datacenter_resize": "Größe ändern"
 }
```
Comment (tibs245) [inline packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/translations/Messages_de_DE.json:13]: Why this edited ?

File: packages/manager/modules/nutanix/src/dashboard/general-info/redeploy/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -29,6 +29,7 @@
   "nutanix_redeploy_custom_config_ip_unique_error": "Cette IP est déjà utilisée sur ce cluster",
   "nutanix_redeploy_custom_config_ip_not_in_cidr_subnet_error": "Les adresses ne sont pas sur le même réseau.",
   "nutanix_redeploy_custom_config_ip_format_error": "Le format de l'IP n'est pas correct",
+  "nutanix_redeploy_custom_config_vlan_id_error": "Le champ VLAN ID doit être un numéro compris entre 0 et 4000",
```
Comment (tibs245) [inline packages/manager/modules/nutanix/src/dashboard/general-info/redeploy/translations/Messages_fr_FR.json:32]: To verify if it's not 1 -> 4094 : https://netseccloud.com/understanding-vlan-id-ranges-what-they-mean-for-your-network

If PO can check it

Comment (darsene) [inline packages/manager/modules/nutanix/src/dashboard/general-info/redeploy/translations/Messages_fr_FR.json:32]: Sounds a bit weird to me, can you check the translation with PO
```suggestion
  "nutanix_redeploy_custom_config_vlan_id_error": "Le champ VLAN ID doit être un nombre compris entre 0 et 4000",
```

**Comment by darsene** [comment]
@ThibaudCrespin can you rebase to fix Jest tests

**Pull request closed without merging**